src/metabase/permissions/models/permissions.clj

```diff
@@ -483,16 +489,17 @@
   [group-or-id :- permissions.path/MapOrID collection-or-id :- permissions.path/MapOrID]
   (check-is-modifiable-collection collection-or-id)
   (when (perms-group/is-tenant-group? group-or-id)
-    (throw (ex-info (tru "Tenant Groups cannot have write access to any collections.") {})))
+    (throw (ex-info (tru "Tenant groups cannot have write access to any collections.") {})))
```

Comment:
Should these say external groups? I've lost track on whether we always call these 'External groups' or Tenant groups.